lib/hanami/generators/app/application.rb.tt

```diff
@@ -205,6 +205,12 @@ module <%= config[:classified_app_name] %>
       # Inline JavaScript is NOT allowed. To enable it, please use:
       # "script-src 'unsafe-inline'".
       #
+      # Javascript eval() and similar methods are NOT allowed. It needs to be enabled
+      # for Javascript frameworks that rely on
+      # such features, such as vue.js. To enable it, please use:
+      # "script-src unsafe-eval'".
```

Comment:
What do you think about wording this as:
```
      # Inline JavaScript is NOT allowed. To enable it, please use:
      # "script-src 'unsafe-inline'".
      #
      # If you need to use JavaScript's eval() and similar functions, 
      # which some frameworks require, please use:
      # "script-src unsafe-eval'".
```
?

I'd prefer to stay away from mentioned Vue.js specifically. The place to handle that is in a guide for Vue.js + Hanami (which is planned).